gitgud/hooks/__init__.py

```diff
@@ -3,5 +3,6 @@
     # ('git-hook-name', 'modulename')
     ('post-rewrite', 'postrewrite'),
     ('commit-msg', 'commitmsg'),
-    ('post-merge', 'postmerge')
+    ('post-merge', 'postmerge'),
```

Comment:
Go back through this branch and do an interactive rebase to get rid of all the code added by post-merge. You can force push to here

Reply:
All features are rebased off track_commits. Track_commits has been modified to hold the definitive versions of shared files.